src/note-components/authentication/auth.css

```diff
@@ -0,0 +1,83 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,300;0,400;0,500;0,900;1,300&display=swap');
+
+body {
+    font-family: 'Montserrat', sans-serif;
```

Comment:
Instead of writing this again and again, you can put this in a separate file and then import it.

Reply:
Okay, will make a root.css